sympy/functions/combinatorial/factorials.py

```diff
@@ -32,8 +32,6 @@ def _eval_simplify(self, **kwargs):
 ###############################################################################
 ######################## FACTORIAL and MULTI-FACTORIAL ########################
 ###############################################################################
-
-
```

Comment:
Why? This change doesn't appear to be relevant for this PR.

Reply:
Done sir. Anything else?